Source/WebCore/platform/graphics/displaylists/DisplayListItems.cpp

```diff
@@ -421,6 +421,19 @@ void ClearRect::apply(GraphicsContext& context) const
     context.clearRect(m_rect);
 }
 
+DrawControlPart::DrawControlPart(const ControlPart& part, const FloatRect& rect, float deviceScaleFactor, const ControlStyle& style)
```

Comment:
Should we drop the `const` here, and avoid the `const_cast`? None of the other display list items are written this way.

Reply:
I added the const qualifier per Simon request in https://github.com/WebKit/WebKit/pull/7566#discussion_r1050011917. It seems it is (and will be) causing trouble. So I removed it.